app/webpacker/packs/stylesheets.scss

```diff
@@ -8,4 +8,4 @@
 @import '../src/stylesheets/user.scss';
 @import '../src/stylesheets/team.scss';
 @import '../src/stylesheets/admin/teamlist.scss';
-@import '../src/stylesheets/custom.scss.less';
```

Comment:
Why was this `.less`? Can I leave it out, as nothing is in less in there. See below in `custom.scss`.

Reply:
I don't know why it's .less. I just migrated everything as it was.

Reply:
Well then, I drop it. The less the better.